### ISSUE TITLE
Version 1.1.2 breaks the parsing

### ISSUE THREAD

**Issue by kakaska** [ovidigital/js-object-to-json#18]
Hi

The changes introduced in the `1.1.2` version breaks the old parsing mechanism.
Here you can find the example code:

```php
<?php

use OviDigital\JsObjectToJson\JsConverter;

include 'vendor/autoload.php';

$input = '{sources:[],image:"http://127.0.0.1/sapsdvcx3kgq.jpg",duration:"3200",width:"100%"}';

var_dump(JsConverter::convertToJson($input));
```

Output with version 1.1.1:
```
string(91) "{"sources":[],"image":"http://127.0.0.1/sapsdvcx3kgq.jpg","duration":"3200","width":"100%"}"
```

Output with version 1.1.2:

```
string(30) "{"sources":[],"image:""http":""
```
```
                                 ^
                                 completly broken
```

I think the `\OviDigital\JsObjectToJson\JsConverter::removeComments()` is not production ready.

**Comment by ovidyou** [comment]
@kakaska thank you for reporting this. I will try to fix this today.

**Comment by ovidyou** [comment]
@kakaska I pushed a fix in version 1.1.4, please try it and let me know if that solved the issue.

**Comment by kakaska** [comment]
Thank you for fast response. Everything is working again with the new release.
